packages/bitcore-wallet-service/src/lib/emailservice.ts

```diff
@@ -351,141 +338,135 @@ export class EmailService {
   }
 
   _readAndApplyTemplates(notification, emailType, recipientsList, cb) {
-    var self = this;
-
     async.map(
       recipientsList,
-      function(recipient, next) {
+      (recipient: any, next) => {
```

Comment:
why any?

Reply:
Removed to use a new 'Recipient' interface